pkg/pullsync/pullsync.go

```diff
@@ -50,6 +50,7 @@ var (
 
 const (
 	storagePutTimeout = 5 * time.Second
+	makeOfferTimeout  = 5 * time.Minute
```

Comment:
My nodes are (well, were before the stampes expired) routinely taking longer than 5 minutes to construct the offer.    makeOffer's context is automatically cancelled at the end of the commit round by the agent, so why have a local timeout specified here?

Reply:
this is puller/pullsync stuff, not reserve sampler :)

Reply:
Duh. makeOffer, not makeSample! That's what I get for commenting before I'm fully awake. I'll blame it on Nicole, the hurricane that passed by us last night!

Reply:
jeez, a bad year for florida this year! so many hurricanes

Reply:
At least Nicole was barely a category 1, just above a tropical storm.   Amazingly, we didn't even lose power; a few blinks overnight, but no outright loss.